### DISCUSSION TITLE
Looking for a partner!

### DISCUSSION THREAD

**Discussion by ThatCodingDonut** [community/community#139296] [GitHub Education]
### Hi everyone,

I was looking for some partners to maybe hop on a group chat. I honestly want some people who can inspire each other and help with fun projects in python or pretty much any other language. :D

You can reach me at @ThatCasualDonut on discord, or corbinchandler777@icloud.com

**Comment by floppek** [comment]
Hey you have friends request disabled on discord add me back floppek.ltc
